Lib/asyncio/tasks.py

```diff
@@ -216,15 +216,13 @@ def cancel(self, msg=None):
         self._cancel_message = msg
         return True
 
-    def cancelling(self):
-        return self._cancel_requested
+    def cancelling(self) -> int:
```

Comment:
@Tinche I know you wish the stdlib used type annotations (and I can't disagree) but I don't think anyone benefits from adding one or two annotations (that won't be checked). The place to change is typeshed.

(FWIW I am less worried about the annotations in timeouts.py -- it's a new, separate file, so at least it's locally consistent. But we still need to add it to typeshed too if anyone besides human readers of the source code here is to benefit.)

Reply:
Roger, I added it to help myself. I will remove it.